tox.ini

```diff
@@ -7,7 +7,7 @@ deps =
    git+https://github.com/PyCQA/astroid@master
    isort
 
-commands = pylint -rn --rcfile={toxinidir}/pylintrc --load-plugins=pylint.extensions.docparams, pylint.extensions.mccabe {envsitepackagesdir}/pylint
+commands = pylint -rn --rcfile={toxinidir}/pylintrc --load-plugins=pylint.extensions.docparams,pylint.extensions.typing {envsitepackagesdir}/pylint
```

Comment:
Because of the spaces mccabe was never actually running.  When I removed the spaces mccabe spewed lots of lint which caused the tests to fail, so I removed it.